Docs: add video generation instructions for xdit-framework images. (#5510)
RadeonOpenCompute/ROCm: docs/how-to/rocm-for-ai/inference/xdit-video-diffusion.rst

.. meta::
   :description: Learn to validate diffusion model video generation on MI300X, MI350X and MI355X accelerators using
                 prebuilt and optimized docker images.
   :keywords: xDiT, diffusion, video, video generation, validate, benchmark

********************
xDiT video inference
********************

.. _xdit-video-diffusion:

.. datatemplate:yaml:: /data/how-to/rocm-for-ai/inference/xdit-inference-models.yaml

    {% set docker = data.xdit_video_diffusion.docker %}
    {% set model_groups = data.xdit_video_diffusion.model_groups%}

    The `amdsiloai/pytorch-xdit Docker <{{ docker.docker_hub_url }}>`_ image offers a prebuilt, optimized environment based on `xDiT <https://github.com/xdit-project/xDiT>`_ for

>     benchmarking diffusion model video generation on gfx942 series (AMD Instinct™ MI300) and

Reviewer: ```suggestion
    benchmarking diffusion model video generation on gfx942 series (AMD Instinct™ MI300X) and
```

Reviewer: Is MI325X also supported?

Reviewer 2: Since we build ROCm for gfx942 all variants are supported (MI300x, MI308, MI325X...) however we currently only tune for MI300X, so the OOB performance for other variants it not optimal. The goal would be to also tune for the other variants in the future. @kTorp maybe we need to detail this explicitly?

>     gfx950 series (AMD Instinct™ MI350 and MI355) accelerators.

Reviewer: ```suggestion
    gfx950 series (AMD Instinct™ MI350X and MI355X) accelerators.
```

                {% else %}
                <div class="col-6 px-2 model-param" data-param-k="model" data-param-v="{{ model.model_name }}" data-param-group="{{ model_group.tag }}" tabindex="0">{{ model.model }}</div>
                {% endif %}
            {% endfor %}
        {% endfor %}
                </div>
            </div>
        </div>

    {% for model_group in model_groups %}
        {% for model in model_group.models %}

    .. container:: model-doc {{model.model_name}}

        To learn more about your specific model see the `{{ model.model }} model card on Hugging Face <{{ model.url }}>`_ 
        or visit the `GitHub page <{{ model.github }}>`_. Note that some models require access authorization before use via an
        external license agreement through a third party.

        {% endfor %}
    {% endfor %}

    Pull the Docker image
    ===================== 

    For this tutorial it's recommended to use the following docker image, which can be pulled
    with the command

    .. code-block:: shell

        docker pull amdsiloai/pytorch-xdit:v25.9

    .. admonition:: Release notes for v25.9

        - Initial release
        - ROCm: 7.0.0rc

>         - Add support for gfx942 series (AMD Instinct™ MI300) and gfx950 series (AMD Instinct™ MI350 and MI355)

Reviewer: ```suggestion
        - Add support for gfx942 series (AMD Instinct™ MI300X) and gfx950 series (AMD Instinct™ MI350X and MI355X)
```
